corporate/models.py

```diff
@@ -32,14 +32,14 @@ def __str__(self) -> str:
         return f"{self.realm!r} {self.stripe_customer_id}"
 
     @property
-    def is_self_hosted(self) -> bool:
+    def is_self_hosted(self) -> bool:  # nocoverage
         is_self_hosted = self.remote_server is not None
         if is_self_hosted:
             assert self.realm is None
         return is_self_hosted
 
     @property
-    def is_cloud(self) -> bool:
+    def is_cloud(self) -> bool:  # nocoverage
```

Comment:
I think I'd prefer to add this file to `not_yet_fully_covered`; at least it's not obvious to me why we would want this to be forever uncovered, which is usually what we use `nocoverage` comments for.

Reply:
Well really someone should write a test for these properties—and until that happens, we want to prevent *additional* uncovered code from being accidentally added to this file, right?

Reply:
Or we can remove these unused properties and replace the constraint they intend to enforce with a true SQL `CONSTRAINT`?

Reply:
> Or we can remove these unused properties and replace the constraint they intend to enforce with a true SQL CONSTRAINT?

Yeah, replacing those with a SQL CONSTRAINT is probably a better answer than writing a test. We could also just write `assert not is_cloud or self.remote_server is None` for these.